client/src/app/api/api/api.service.ts

```diff
@@ -213,6 +225,39 @@ export class ApiService {
     );
   }
 
+  private postLabel(endpoint: string, idInstance: string, labelName: string, observe: any = 'body', reportProgress: boolean = false): any {
```

Comment:
why did you create two methods for the post request for adding new labels? As I gather from their name they are not really reusable in any place since they are specifically made for the purpose of adding labels, so I don't quite understand the splitting here.